adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/batch-delete-rum-metric-definitions.rst

**To remove the specified metrics from being sent to an extended metrics destination**

> The following ``batch-delete-rum-metric-definitions`` removes the specified metrics from being sent to an extended metrics destination. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``batch-delete-rum-metric-definitions`` example removes the specified metrics from being sent to an extended metrics destination. ::
```

> 	aws rum batch-delete-rum-metric-definitions \

Reviewer: All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--app-monitor-name AWSApp \
		--destination Evidently \
		--destination-arn arn:aws:evidently:us-east-1:123456789012:project/petfood/feature/petfood-upsell-text \

> 		--metric-definition-ids 77gh6a55-bh77-5eed-a338-d8750b544a2

Reviewer: ```suggestion
        --metric-definition-ids 77gh6a55-bh77-5eed-a338-d8750b544a2

This command produces no output.
```
